src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -14,19 +14,21 @@ namespace Azure.ApplicationModel.Configuration.Tests
     [Category("Live")]
     public class ConfigurationLiveTests
     {
-        static readonly ConfigurationSetting s_testSetting = new ConfigurationSetting(
-            string.Concat("key-", Guid.NewGuid().ToString("N")),
-            "test_value"
-        )
+        private ConfigurationSetting CreateSetting()
         {
-            Label = "test_label",
-            ContentType = "test_content_type",
-            Tags = new Dictionary<string, string>
+            return new ConfigurationSetting()
             {
-                { "tag1", "value1" },
-                { "tag2", "value2" }
-            }
-        };
+                Key = string.Concat("key-", Guid.NewGuid().ToString("N")),
```

Comment:
What about extracting `string.Concat("key-", Guid.NewGuid().ToString("N"))` into a helper method `GenerateKeyId()` seems to be used quite a bit in this file.